src/python/e2e-test-runner/e2e_test_runner/test_main.py

```diff
@@ -23,8 +18,7 @@
 GqlLensDict = Dict[str, Any]
 
 
-@pytest.mark.integration_test
```

Comment:
[dunno if this will screw things up or not, seeing in CI. I believe this might be a leftover from pre-pants, and this won't affect anything since this isn't a `python_tests()` target.]

Reply:
https://github.com/grapl-security/grapl/blob/7d67105cb6a29dbdfe668cc311ccff811e3434f7/Makefile#L231-L237

Reply:
sicne this is a lambda, not a python_tests(), i think we're good